pkg/k8s/operations.go

```diff
@@ -333,6 +344,148 @@ func ApplyYaml(params ApplyYamlPrams, kubeconfig string, isLocal bool) (output s
 	return outStr, nil
 }
 
+// UpgradeInfra upgrades the Chaos Infrastructure using the provided manifest and kubeconfig with the help of client-go library.
+
+func UpgradeInfra(manifest []byte, kubeconfig string) (string, error) {
+
+	// Get Kubernetes and dynamic clients along with the configuration.
+
```

Comment:
please remove the intermittent newline between the comment and the code at all places.
```suggestion
```